tedana/utils/utils.py

```diff
@@ -106,6 +107,87 @@ def load_image(data):
     return fdata
 
 
+def load_data2(data, n_echos=None):
```

Comment:
Is this being used anywhere, at this point ?

Reply:
It would replace `load_data` as the input function for multi-echo data files.

Reply:
Got it, thanks !! Can we drop the gifti support in there too, then ?